packages/cw-schema-codegen/templates/python/struct.tpl.py

```diff
@@ -0,0 +1,32 @@
+# This code is @generated by cw-schema-codegen. Do not modify this manually.
+
+import typing
+import decimal
+from pydantic import BaseModel, RootModel, Base64Bytes
+
+
+{% match ty %}
+{% when TypeTemplate::Unit %}
+class {{ name }}(RootModel):
+    """{% for doc in docs %}
+    {{ doc }}
+    {% endfor %}"""
+    root: None
+{% when TypeTemplate::Tuple with (types) %}
+class {{ name }}(RootModel):
+    """{% for doc in docs %}
+    {{ doc }}
+    {% endfor %}"""
+    root: typing.Tuple[{{ types|join(", ") }}]
+{% when TypeTemplate::Named with { fields } %}
+class {{ name }}(BaseModel):
+    """{% for doc in docs %}
+    {{ doc }}
+    {% endfor %}"""
+    {% for field in fields %}
+    {{ field.name }}: {{ field.ty }}
```

Comment:
Ultimately we need some sort of handling for keywords here. E.g. the `crypto-verify` contract uses `from` as a field name, which causes invalid python code to be generated here.